vignettes/articles/wrapping-apis.Rmd

```diff
@@ -501,13 +501,13 @@ You can make this approach a little more user friendly by providing a helper tha
 ```{r}
 set_api_key <- function(key = NULL) {
   if (is.null(key)) {
-    key <- askpass::askpass("Please enter your API key")
+    key <- rstudioapi::askForPassword("Please enter your API key")
   }
   Sys.setenv("NYTIMES_KEY" = key)
 }
 ```
 
-Using askpass (or similar) here is good practice since you don't want to encourage the user to type their secret key into the console, as mentioned above.
+Using `rstudioapi` (or similar) here is good practice since you don't want to encourage the user to type their secret key into the console, as mentioned above.
```

Comment:
Still valid advice, but then we partially erode the security benefits by immediately calling `Sys.setenv()` 🤔 
https://github.com/rstudio/rsconnect/pull/1027#issuecomment-1820901085

In any case, if we decide to use `rstudioapi` over `askpass`, feels prudent to remove usage of `askpass` here (thereby dropping it from `Suggests` as well).

Reply:
Calling `Sys.setenv()` this was is fine, because it won't end up in the `.Rhistory` (because it's both inside a function and even if called at the top level it'd only record `key`, not the actual value).